src/autowiring/observable.h

```diff
@@ -72,8 +72,18 @@ class observable {
     onChanged();
     return *this;
   }
-};
 
+  bool operator==(const T& rhs) { return val == rhs; }
+  bool operator!=(const T& rhs) { return val != rhs; }
+  bool operator<(const T& rhs) { return val < rhs; }
+  bool operator<=(const T& rhs) { return val <= rhs; }
+  bool operator>(const T& rhs) { return val > rhs; }
+  bool operator>=(const T& rhs) { return val >= rhs; }
+  auto operator*(const T& rhs) -> decltype(val * rhs) { return val * rhs; }
+  auto operator/(const T& rhs) -> decltype(val * rhs) { return val / rhs; }
+  auto operator+(const T& rhs) -> decltype(val * rhs) { return val + rhs; }
+  auto operator-(const T& rhs) -> decltype(val * rhs) { return val - rhs; }
+};
```

Comment:
These operators don't seem to be hit at all via the unit test. I mean, I can remove them and the test still compiles and runs.

I suspect it's because `operator const T&(void) const { return val; }` already does everything we need.

Still investigating.